flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyBufferPool.java

```diff
@@ -61,11 +61,11 @@
 
 	/**
 	 * Arenas allocate chunks of pageSize << maxOrder bytes. With these defaults, this results in
-	 * chunks of 16 MB.
+	 * chunks of 4 MB.
```

Comment:
There are also many other places mentioned `16 MB` in comments, and we should still adjust them.